### ISSUE TITLE
Feature: Host key fingerprint enumeration/gathering

### ISSUE THREAD

**Issue by 0x27** [arthepsy/ssh-audit#19]
A useful feature to have would be enumerating and grabbing the hostkeys/fingerprints from the remote SSH server and storing them as part of the output.

Something similar to what I did here, maybe? https://github.com/0x27/ssh_keyscanner

**Comment by arthepsy** [comment]
I completely agree and this is **already planned**. Currently, it's implemented for SSH1 (see screenshot in #12), but it's missing some details for SSH2, because it requires successful key-exchange.

**Comment by jtesta** [comment]
@0x27 This feature has been implemented in my v2.0.0 release: https://github.com/jtesta/ssh-audit/releases/tag/v2.0.0
